resources/views/WelcomePage/WelcomePage.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import {PageHeader, Card, Row, Col, Divider} from 'antd'
+import {Link} from 'react-router-dom'
+
+const WelcomePage = () => (
+  <>
+    <div style={{}}>
+      <PageHeader
+        className='site-page-header'
+        title='Welcome to Cycl.io' />
+    </div>
+    <Divider>Overview</Divider>
+
+    <Row justify='space-around'>
+      <Col span={8}>
```

Comment:
I don't like the approach of having 3 cards without any space in between, let's add grid gutter
https://ant.design/components/grid/